benchmarks/benchmarks/ci/esmf_regridder.py

```diff
@@ -84,3 +88,58 @@ def setup(self, type):
 
     def time_perform_regridding(self, type):
         _ = self.regridder(self.src)
+
+
+class TimeLazyRegridding:
+    # Prevent repeat runs between setup() runs - data won't be lazy after 1st.
+    number = 1
+    # Compensate for reduced certainty by increasing number of repeats.
+    repeat = (10, 10, 10.0)
+    # Prevent ASV running its warmup, which ignores `number` and would
+    # therefore get a false idea of typical run time since the data would stop
+    # being lazy.
+    warmup_time = 0.0
+
+    def setup_cache(self):
+        SYNTH_DATA_DIR = Path().cwd() / "tmp_data"
```

Comment:
I assume that `asv` deletes the current working directory at the end of the run so there is no need to clean up after ourselves here?

Reply:
I believe so, see this comment https://github.com/airspeed-velocity/asv/issues/511#issuecomment-300502282.